docs/_reference/natural_language_actions_api.md

```diff
@@ -5,21 +5,22 @@ layout: post-toc
 redirect_from: /zapier_products/natural-language-actions-api
 ---
 
-# Natural Language Actions API
+# AI Actions
 
-Zapier’s [Natural Language Actions (NLA) API](https://nla.zapier.com/docs/) is an AI alpha product designed to work with natural language-based products. It leverages the Zapier platform, with over [6000 apps](https://zapier.com/apps). You can include the capabilities of Zapier’s platform in your own product.
+Zapier’s [AI Actions](https://actions.zapier.com/) is an AI alpha product designed to work with natural language-based products. It leverages the Zapier platform, with over [6000 apps](https://zapier.com/apps). You can include the capabilities of Zapier’s platform in your own product.
 
 With the Natural Language Actions API, you can:
-* Access Zapier’s platform of 6000+ apps within your own product.
-* Integrate with chatbots or [large language models](https://en.wikipedia.org/wiki/Wikipedia:Large_language_models).
-* Power any integration project.
 
-Learn more in our [Natural Language Actions API documentation](https://nla.zapier.com/docs/).
+- Access Zapier’s platform of 6000+ apps within your own product.
+- Integrate with chatbots or [large language models](https://en.wikipedia.org/wiki/Wikipedia:Large_language_models).
```

Comment:
Okay yikes, we need to change this wikipedia page. Go ahead and read the first 3 paragraphs. I recommend using this link instead: https://en.wikipedia.org/wiki/Large_language_model

Reply:
Done!